slack-api-client/src/main/java/com/slack/api/methods/impl/AsyncMethodsClientImpl.java

```diff
@@ -2873,4 +2875,14 @@ public CompletableFuture<WorkflowsUpdateStepResponse> workflowsUpdateStep(Reques
         return workflowsUpdateStep(req.configure(WorkflowsUpdateStepRequest.builder()).build());
     }
 
+    @Override
+    public CompletableFuture<EntityPresentDetailsResponse> entityPresentDetails(EntityPresentDetailsRequest req) {
+        return executor.execute(ENTITY_PRESENT_DETAILS, toMap(req), () -> methods.entityPresentDetails(req));
+    }
+
+    @Override
+    public CompletableFuture<EntityPresentDetailsResponse> entityPresentDetails(RequestConfigurator<EntityPresentDetailsRequest.EntityPresentDetailsRequestBuilder> req) {
+        return entityPresentDetails(req.configure(EntityPresentDetailsRequest.builder()).build());
+    }
```

Comment:
📠 nit: Personal preference for this section to move up a few lines for alphabetics-